### PR TITLE
Revert "fix ejabberd_logleverl_SUITE"

### PR DESCRIPTION
**Pull request by michalwski**
This reverts commit 53cb243fbd7e234c536ddc3acb7a634b16a144a1 and fixes the suite for Erlang/OTP 21 in other way.

The current fix sets the default loglevel for Erlang's built-in logger to `info` where progress reports are visible.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[4996.1](https://travis-ci.org/esl/MongooseIM/jobs/398166573) / Erlang 19.3 / small_tests / 17c549a4f05c75fd5d3781dc4465478ad1d581bb
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/1953/4996/small_tests.19.3)

**Comment by mongoose-im** [comment]
[4997.1](https://travis-ci.org/esl/MongooseIM/jobs/398175278) / Erlang 19.3 / small_tests / 95a225477b815db8f1ddc5f42856f898b9028339
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/1953/4997/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/1953/4997/small_tests.19.3/small/ct_run.mongooseim@localhost.2018-06-29_10.09.21/lib.mongooseim.logs/run.2018-06-29_10.09.22/suite.log.html)

File: test/ejabberd_loglevel_SUITE.erl [hunk 1]
```diff
@@ -17,6 +17,22 @@ all() ->
...
+init_per_suite(Config) ->
+    try
+        %% This is a workaround for Erlang 21 where by default
+        %% progress reports are not visible so the log file is filled with new
+        %% content after lager backend is started.
+        %% We need this in tests in order to reliably wait for the backend start
+        LoggerConfig = logger:get_primary_config(),
+        logger:set_primary_config(LoggerConfig#{level := info})
+    catch _E:_R ->
+              ok
```
Comment (fenek) [inline test/ejabberd_loglevel_SUITE.erl]: I'd add `ct:pal` here, just in case of a failure on OTP 21.

**Comment by mongoose-im** [comment]
[5007.1](https://travis-ci.org/esl/MongooseIM/jobs/399099402) / Erlang 19.3 / small_tests / 673ddda85716a91133bf67439be499b5be094418
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/1953/5007/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/1953/5007/small_tests.19.3/small/ct_run.mongooseim@localhost.2018-07-02_12.44.16/lib.mongooseim.logs/run.2018-07-02_12.44.17/suite.log.html)

**Pull request merged**